tests/run-with-compiler/i3823-b.scala

```diff
@@ -5,6 +5,6 @@ object Test {
       val z: $t = $x
     }
     implicit val toolbox: scala.quoted.Toolbox = scala.quoted.Toolbox.make(getClass.getClassLoader)
-    println(f('{2})(Type.IntTag).show)
+    println(show(f('{2})(Type.IntTag)))
```

Comment:
It seems to me for staging, having `Expr[T].show` will make debugging way easier.

Reply:
Yes, that one is supported as well. In this case I used the variant of show to minimize the diff.

The test could also have been rewritten as

```scala
implicit val toolbox: scala.quoted.Toolbox = scala.quoted.Toolbox.make(getClass.getClassLoader)
run {
  println(f('{2})(Type.IntTag).show)
  '{}
}
```

or 
```scala
implicit val toolbox: scala.quoted.Toolbox = scala.quoted.Toolbox.make(getClass.getClassLoader)
println(run {
  f('{2})(Type.IntTag).show.toExpr
})
```

Reply:
It seems a little unnatural that we need to `run` in order to show expression.

Reply:
We need to be inside the macro `${ ... }` or the `run { ... }` that generated the expression in the first place. The only usecase of using `show` without a `run` of a `$` splice is in tests.